httpclient/src/main/java/org/apache/http/impl/auth/GGSSchemeBase.java

```diff
@@ -258,6 +274,25 @@ public Header authenticate(
         }
     }
 
+    /**
```

Comment:
With so many new public methods we really ought to move all these changes to the 4.6.x branch.

Reply:
I have no problem with moving these changes to 4.6.x.  I'll wait to see if you want anything changed above first.